ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/LogWithMetadata.java

```diff
@@ -1,6 +1,5 @@
 /*
- *
- * Copyright ConsenSys AG.
+ * Copyright contributors to Hyperledger Besu.
```

Comment:
some of these are changing from consensys to hyperledger

Reply:
good find ...